src/components/WalletPopup/SendToken/index.tsx

```diff
@@ -232,18 +264,14 @@ export default function SendToken({
 
         <WarningBrave token={currencyIn} />
 
-        {estimateGas && (
-          <RowBetween>
-            <Label>
-              <Trans>Gas Fee</Trans>
-            </Label>
-            <Label>
-              {estimateGas
-                ? `~ ${formatNumberWithPrecisionRange(estimateGas, 0, 10)} ${NativeCurrencies[chainId].symbol}`
-                : '-'}{' '}
-            </Label>
-          </RowBetween>
-        )}
+        <RowBetween>
+          <Label>
+            <Trans>Gas Fee</Trans>
+          </Label>
+          <Label color={theme.text}>
+            {estimateGas && usdPrice ? `~ ${formattedNum((estimateGas * usdPrice).toString(), true)} ` : '-'}
```

Comment:
`estimateGas * usdPrice` => this should be price of native token right? not usdPrice of token in

Reply:
nice catch. updated